OOVPA.h

```diff
@@ -70,7 +70,7 @@ typedef struct _OOVPA
 typedef struct _LOVP
 {
     unsigned short Offset;
-    unsigned char Value;
+    unsigned short Value;
```

Comment:
The comment is no longer true, since there is no longer a swap between an 8-bit and 16-bit offset, so restoring the original comment is pointless.

A new comment could be added to explain that both values are 16-bit to support large XRefs, however, I'll do that.

Reply:
Yes, re-wording is obviously required, thanks.

Reply:
This was done in the latest commit